ecmascript/codegen/src/lib.rs

```diff
@@ -184,7 +184,9 @@ impl<'a> Emitter<'a> {
         for specifier in &node.specifiers {
             match specifier {
                 ImportSpecifier::Named(ref s) => {
-                    specifiers.push(s);
+                    if !s.is_type_only {
```

Comment:
This seems wrong.
Type-only specifiers should be stripped out by `typescript::strip()`